provisioning/common/etcd/values.yaml

```diff
@@ -105,7 +108,7 @@ startupProbe:
   periodSeconds: 5
   timeoutSeconds: 5
   successThreshold: 1
-  failureThreshold: 60
+  failureThreshold: 300
 livenessProbe:
   initialDelaySeconds: 0 # replaced by startupProbe
```

Comment:
From the incidents we have dealt with in the past, we know that we need to increase this probe timeout.